po/es.po

```diff
@@ -15159,27 +15161,27 @@ msgstr "cambiar el orden de las imágenes"
 
 #: ../src/views/lighttable.c:4632
 msgid "zoom all the images"
-msgstr "Ampliar todas las imágenes"
+msgstr "ampliar a la vez todas las imágenes"
 
 #: ../src/views/lighttable.c:4637
 msgid "pan inside all the images"
-msgstr "Desplazarse dentro de todas las imágenes"
+msgstr "desplazar a la vez todas las imágenes"
 
 #: ../src/views/lighttable.c:4643
 msgid "zoom current image"
-msgstr "ampliar la imagen actual"
+msgstr "ampliar la imagen designada"
 
 #: ../src/views/lighttable.c:4649
 msgid "pan inside current image"
-msgstr "desplazarse dentro de la imagen actual"
+msgstr "desplazarse por la imagen designada"
```

Comment:
Same as before, "designada" is not commonly used for this meaning, I prefer "actual".

Reply:
"actual" loses its meaning in this context, as well as being vague. That's why the other word.